src/dune_rules/lib.ml

```diff
@@ -2092,39 +2180,3 @@ let to_dune_lib
   in
   Dune_package.Lib.of_dune_lib ~info ~main_module_name
 ;;
-
-module Local : sig
```

Comment:
This module was just moved up to make `Local` available in scope for `get_compile_info`.